src/main/java/pl/pjwstk/woloappapi/controller/EventController.java

```diff
@@ -0,0 +1,44 @@
+package pl.pjwstk.woloappapi.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.pjwstk.woloappapi.model.Event;
+import pl.pjwstk.woloappapi.service.EventService;
+import pl.pjwstk.woloappapi.utils.EventNotFoundException;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/event")
+public class EventController {
+
+    private final EventService eventService;
+
+    @GetMapping("/all")
+    public ResponseEntity<List<Event>> getEvents(){
+        List<Event> events = eventService.getAllEvents();
+        return new ResponseEntity<>(events, HttpStatus.OK);
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Event> getEventById(@PathVariable Long id){
+        try {
```

Comment:
to bym wyniósł do osobnej klasy łapiącej odpowiednie exceptiony i mapującej to na właściwe ciało response'a.

Tu przykład takiej klasy:
https://github.com/hmcts/div-case-orchestration-service/blob/master/src/main/java/uk/gov/hmcts/reform/divorce/orchestration/controller/advice/GlobalExceptionHandler.java#L22